react-hs-examples/redux/build.sh

```diff
@@ -0,0 +1,6 @@
+stack setup --stack-yaml stackjs.yaml
```

Comment:
instead of having a local build script, could you add this to /scripts/build.sh?

i admit that this has the drawback that the package is less self-contained.  (so alternatively, we could keep this and call it from /scripts/build.sh, and change the TODO example and the other places with this problem accordingly.)

in any case, the build script should be run by the ci.